packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -207,6 +219,44 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
         _stake(_from, _from, _amount, _data);
     }
 
+    /**
+    * @notice Lock `@tokenAmount(self.token(), _amount)` of sender's active balance
+    * @param _lockManager Address of the lock manager that will control the lock
+    * @param _amount Amount of active tokens to be locked
+    */
+    function lockActivation(address _lockManager, uint256 _amount) external {
+        _lockActivation(msg.sender, _lockManager, _amount);
+    }
+
+    /**
+    * @notice Unlock `@tokenAmount(self.token(), _amount)` of `_guardian`'s active balance
+    * @param _guardian Address of the guardian unlocking the active balance of
+    * @param _lockManager Address of the lock manager controlling the lock
+    * @param _amount Amount of active tokens to be unlocked
+    * @param _deactivate Whether the requested amount must be deactivated too
+    */
+    function unlockActivation(address _guardian, address _lockManager, uint256 _amount, bool _deactivate) external {
+        bool canUnlock = _lockManager == msg.sender || ILockManager(_lockManager).canUnlock(_guardian, _amount);
+        require(canUnlock, ERROR_CANNOT_UNLOCK_ACTIVATION);
+
+        ActivationLocks storage activationLocks = guardiansByAddress[_guardian].activationLocks;
+        uint256 lockedAmount = activationLocks.lockedBy[_lockManager];
+        require(lockedAmount > 0, ERROR_ZERO_LOCK_ACTIVATION);
+        require(_amount > 0, ERROR_UNLOCK_ACTIVATION_ZERO);
```

Comment:
I wonder if we should change this so it acts more like the deactivation?

If we do the `canUnlock()` check after this, could we have `amount = lockedAmount` if given 0?